Build-Debug.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
+
+call "%~dp0tools\Build-Debug-Server" || goto :error
+
+goto :done
+
+:error
+echo Build failed with error %errorlevel%.
+
+:done
+if not defined IGNORE_PAUSE pause
+exit /b %errorlevel%
```

Comment:
Please use:

```sh
@call "%~dp0tools\Run" Build Debug x pause
```

(The rest is an artifact of client build system without the client build system).